src/Criterion.cpp

```diff
@@ -53,19 +40,24 @@ float Criterion::getWeight() const { return weight_; }
 
 void Criterion::setWeight(float weight) { weight_ = weight; }
 
+void Criterion::getRandomCriterionWeight(bool changeSeed) {
```

Comment:
This is present since we didn't merge Emma's branch and did a random generator.
I want to have a seed controllable random generator (we need to implement it). 
Added a task in troll

Reply:
trello